ext/intl/timezone/timezone.stub.php

```diff
@@ -112,6 +112,14 @@ public function getErrorMessage(): string|false {}
      */
     public static function getGMT(): IntlTimeZone {}
 
+#if U_ICU_VERSION_MAJOR_NUM >= 74
+    /**
+     * @tentative-return-type
+     * @alias intltz_get_iana_id
+     */
+    public static function getIanaID(string $timezoneId): string|false {}
```

Comment:
I'm not sure what the best formatting would be, but the method name mixes two formatting styles for abbreviations: where only the first character is uppercase (`Iana`) vs. where all characters are (`ID`).

So maybe this name could be used?
```suggestion
    public static function getIanaId(string $timezoneId): string|false {}
```

(I see that this suggestion is inconsistent with the `getEquivalentID()` method name. :/)

Reply:
Yes I tried to be somewhat consistent with the rest, getIANAID looked "too much" for me.

Reply:
@TimWolla has opinions about casing with abbreviations

Reply:
I do. *Technically* the current policy is that the method needs to be called `getIANAID()`. Because I find that dumb, I created a ML thread about this: https://externals.io/message/120959#120959. I need to turn that into an RFC still.

That said, I believe that `getIanaID()` is the useful middle-ground between consistency and readability here.